### PR TITLE
Add arrow buttons

### PR DESCRIPTION
**Pull request by georgec12104531**

### CODE REVIEW

File: frontend/src/components/videos/videos_index.jsx [hunk 1]
```diff
@@ -5,6 +5,13 @@ import Loader from "react-loader-spinner";
 import VideosIndexItem from "./videos_index_item";
 import "./animate.css";
 
+import RightArrow from "./right-arrow.png";
+import LeftArrow from "./left-arrow.png";
+import Space from "./space-btn.png";
+
+const FIREBASE_VIDEO_URL =
```
Comment (JOlivier92) [inline frontend/src/components/videos/videos_index.jsx]: delete unused variable in file

File: frontend/src/components/videos/videos_index.jsx [hunk 2]
```diff
@@ -91,6 +98,28 @@ class VideosIndex extends React.Component {
...
+              key={video.id}
+              firebaseURL={
+                "https://firebasestorage.googleapis.com/v0/b/crushd-efd3f.appspot.com/o/" +
+                video.videoURL +
+                "?alt=media&token=d2acb0e3-28d7-4f43-b43e-dfd7bb3c1ae9"
+              }
+            />
+          ))}
+          <div className="arrow">
+            <img className="space-btn" src={Space} />
```
Comment (JOlivier92) [inline frontend/src/components/videos/videos_index.jsx]: add alt="spacebtn" inside of img

File: frontend/src/components/videos/videos_index.jsx [hunk 3]
```diff
@@ -91,6 +98,28 @@ class VideosIndex extends React.Component {
...
+            &nbsp;&nbsp;
+            <p className="arrow-text">MUTE/UNMUTE SOUND</p>
+            &emsp;
+            <img className="keyboard-btn" src={LeftArrow} />
```
Comment (JOlivier92) [inline frontend/src/components/videos/videos_index.jsx]: add alt="left-arrow" inside of img

File: frontend/src/components/videos/videos_index.jsx [hunk 4]
```diff
@@ -91,6 +98,28 @@ class VideosIndex extends React.Component {
...
+            <p className="arrow-text">NEXT PROFILE</p>
+            &emsp;
+            <img className="keyboard-btn" src={RightArrow} />
```
Comment (JOlivier92) [inline frontend/src/components/videos/videos_index.jsx]: add alt="right-arrow" inside of img

File: frontend/src/components/videos/videos_index.jsx (hunk 1 above)
Comment (georgec12104531) [reply frontend/src/components/videos/videos_index.jsx]: hmm, I use the variables in the file though..

**Pull request merged**